src/JavaDependenciesManager.ts

```diff
@@ -17,26 +17,25 @@
 import * as fs from 'fs';
 import * as path from 'path';
 import * as vscode from 'vscode';
-import * as kamelCli from './kamel';
 import * as utils from './CamelKJSONUtils';
+import { execSync } from 'child_process';
 
 const PREFERENCE_KEY_JAVA_REFERENCED_LIBRARIES = "java.project.referencedLibraries";
 export const CAMEL_VERSION = "3.20.1";
 
 export async function initializeJavaDependenciesManager(context: vscode.ExtensionContext): Promise<void> {
-	const destination = parentDestinationFolderForDependencies(context);
+	const destination = destinationFolderForDependencies(context);
 	await downloadCommonCamelKJavaDependencies(context);
-	await initializeJavaSettingManagement(path.join(destination, 'dependencies'));
+	await initializeJavaSettingManagement(destination);
 }
 
 async function downloadJavaDependencies(context: vscode.ExtensionContext, groupId: string, artifactId: string, version: string) {
-	const destination = parentDestinationFolderForDependencies(context);
+	const destination = destinationFolderForDependencies(context);
 	const pomTemplate = context.asAbsolutePath(path.join('resources', 'maven-project', 'pom-to-copy-java-dependencies.xml'));
-	const directDependenciesDestination = path.join(destination, 'dependencies');
-	fs.mkdirSync(directDependenciesDestination, { recursive: true });
+	fs.mkdirSync(destination, { recursive: true });
 
 	const mvn = require('maven').create({
```

Comment:
potential improvement: remove completely maven for initial setup and use jbang with an inlined route